src/main/java/domain/lotto/LottoBundle.java

```diff
@@ -1,21 +1,63 @@
 package domain.lotto;
 
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.List;
+import domain.result.LottoRank;
+import domain.result.LottoResult;
+import domain.result.WinningResult;
+
+import java.util.*;
+import java.util.stream.Collectors;
 
 public class LottoBundle {
     private final List<Lotto> lottoBundle;
 
-    public LottoBundle(List<Lotto> lottoBundle) {
+    private LottoBundle(final List<Lotto> lottoBundle) {
         this.lottoBundle = new ArrayList<>(lottoBundle);
     }
 
+    public static LottoBundle of(final List<List<Integer>> lottoNumberBundle) {
+        return new LottoBundle(lottoNumberBundle.stream()
+                .map(numbers -> Lotto.of(numbers))
+                .collect(Collectors.toList()));
+    }
+
+    public LottoResult checkResult(final WinningResult winningResult) {
+        final Map<LottoRank, Integer> lottoResult = new HashMap<>();
+        Arrays.stream(LottoRank.values())
+                .forEach(lottoRank -> lottoResult.put(lottoRank, 0));
+
+        for (Lotto lotto : lottoBundle) {
+            LottoRank lottoRank = checkSingleLottoRank(lotto, winningResult);
+            lottoResult.put(lottoRank, lottoResult.get(lottoRank) + 1);
+        }
```

Comment:
```suggestion
        for (Lotto lotto : lottoBundle) {
            LottoRank lottoRank = checkSingleLottoRank(lotto, winningResult);
            lottoResult.put(lottoRank, lottoResult.getOrDefault(lottoRank, 0) + 1);
        }
```

getOrDefault를 통해 조금 더 간단하게 만들 수 있을 것 같습니다

Reply:
해당 방식을 넣어주니 checkSingleLottoRank에서 반환이 안된 LottoRank같은 경우, 테스트 코드와 outputView에서 접근하려 할때 NullPointerException을 발생시켰어요.
그에 따라 에러가 나는 모든 로직을 getOrDefault로 변경하였습니다. 

getOrDefault를 도입하고 느낀 장점은, 
실제 결과 값으로 반환되는 것들만 Map의 원소로 넣어줘서 Map 자체가 관리해야 할 데이터의 양이 줄었다는 것이였어요. 
현재로써는 LottoRank에 6가지 항목밖에 없어서, Map의 작은 사이즈로의 관리 필요성을 크게 못느껴 get을 썼던것 같아요. 
나중에 큰 DB를 다룰때에는 getOrDefault를 쓰는게 좋겠다 생각이 들었어요.